### ISSUE TITLE
[Bug] Different Nav bars for different pages

### ISSUE THREAD

**Issue by PranavSikarwal** [SurajPratap10/Imagine_AI#160]
**Describe the bug**
1- We use ejs to include the nav bar so that nav bar remains same throught the project but through a PR header includes were removed and a new nav bar was added.

2- Previously a highlight nav bar item feature was present which made particular page visited to be highlighted when visited.
![image](https://github.com/SurajPratap10/Imagine_AI/assets/85329574/e98cd75e-6ef9-4b53-8e51-a78d3cd5b975)

**To Reproduce**
1. Again inject the include of header in home.ejs

**Expected behavior**
Same lengths of nav in all the pages.

WE can notice the different lengths of nav bars

home nav bar
![image](https://github.com/SurajPratap10/Imagine_AI/assets/85329574/860524e7-ac3e-4459-baf9-1119f5cbae7d)

other nav bar
![image](https://github.com/SurajPratap10/Imagine_AI/assets/85329574/713e6fba-4b95-4c68-9cd8-c67f3ec52d21)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome]

**Comment by 9520prashant** [comment]
It's not an issue bro we designed website in such way

**Comment by PranavSikarwal** [comment]
@9520prashant  if there is a header include available than why to manually write header, this creates redundancy and you can never edit the navigation bar in both the pages simultaneously.
That was the sole reason i have used a templating engine,  for that purpose only.
and then talking about the path to be active 
it is always a good practice to tell the user about which page they are upon.
and the functionality is still present just it broke.
So @9520prashant it certainly is a bug, as the code for the feature broke, whereas still code is present.

**Comment by 9520prashant** [comment]
Yes Bro you are right @PranavSikarwal but Yesterday I have already redesigned the navbar , but i don't know why the change are not merged @SurajPratap10

**Comment by PranavSikarwal** [comment]
@9520prashant So why are you telling my issue to be wrong, i cannot find the reason for if.
Please maintain good etiquettes while involved in a community, this is a request. and do not spam here please refer discord channel or connect to me on discord: PЯДИДV#6001
Request: please do not spam in a issue, requesting you again.

**Comment by 9520prashant** [comment]
I am not spamming bro  and it is [See our design](https://www.figma.com/file/a8pdJhaVBV13EQZtRTovod/Untitled?type=design&node-id=1%3A2&t=oY9nY1kscoQC99FJ-1)

and bro I humbly said that it's not issue we have designed the website. @PranavSikarwal and you can confirm it from @SurajPratap10

**Comment by PranavSikarwal** [comment]
@9520prashant what can i do about it, i am just saying that it is important to use includes as they remove redundancy, so as there is no include used and no active navigation shown you can basically see.
Please do not ping here just talk about technologies here, i request you
@SurajPratap10 will answer for the issue, as i have provided all the information about it, please @9520prashant you are not the  project admin nor a mentor.
